com.jianguoyun.Nutstore.json

```diff
@@ -20,7 +20,7 @@
 				"install -D -t /app/bin -m 755 nutstore",
 				"install -D -t /app/bin -m 755 apply_extra",
 				"install -D -t /app/share/applications -m 644 com.jianguoyun.Nutstore.desktop",
-				"install -D -t /app/share/icons/hicolor/128x128/apps -m 644 com.jianguoyun.Nutstore.png",
```

Comment:
We require a 128px icon

Reply:
But this software only provides a 64x64 icon, should I scale it to 128x128?

Reply:
Yes, please. Keep the 64px version as well